test/printer_test.rb

```diff
@@ -72,6 +72,22 @@ def setup
     print_stuff
   end
 
+  should "return an array of jobs" do
+    stub_connection
+    fake_connection.expects(:post).with('/jobs', {:printerid => "printer"}).returns(real_jobs_hash)
+
+    print_job_array = get_all_jobs
+
+    assert_equal(
+      [true, true, true, true],
+      [print_job_array.is_a?(Array),
+       print_job_array.map(&:class).uniq.length.equal?(1),
+       print_job_array.map(&:class).uniq.first.equal?(CloudPrint::PrintJob),
+       print_job_array.length.equal?(3)
+      ]
```

Comment:
This is not at all what I meant. Sorry, maybe I should have been more explicit. you only need one assertion because you can test against the real data. I didn't mean for you to fold 4 assertions into one. I meant for you to remove all those redundancies by being more explicit, not less.